Common/Python/PandasConverter.cs

```diff
@@ -135,18 +135,12 @@ public PyObject GetIndicatorDataFrame(IDictionary<string, List<IndicatorDataPoin
 
                 foreach (var kvp in data)
                 {
-                    var index = new List<DateTime>();
-                    var values = new List<double>();
-
-                    foreach (var item in kvp.Value)
-                    {
-                        index.Add(item.EndTime);
-                        values.Add((double)item.Value);
-                    }
-                    pyDict.SetItem(kvp.Key.ToLowerInvariant(), _pandas.Series(values, index));
+                    var index = kvp.Value.Select(x => x.EndTime);
+                    var values = kvp.Value.Select(x => x.Value);
+                    pyDict.SetItem(kvp.Key.ToLowerInvariant(), _pandas.Series(data: values, index: index));
                 }
 
-                return _pandas.DataFrame(pyDict, columns: data.Keys.Select(x => x.ToLowerInvariant()).OrderBy(x => x));
+                return _pandas.DataFrame.from_dict(pyDict);
```

Comment:
`columns: data.Keys.Select(x => x.ToLowerInvariant()).OrderBy(x => x)` the `.OrderBy` is now missing? I see we already set the key to lower above so that part is okay. Suggest avoid these changes if unrequited?

Reply:
Yeah I actually wanted to test that out and see if it mixed up the columns. Turns out it doesn't so there I went ahead and added it back. I can revert to the original if you like. Yesterday when I was digging for the problem I was poking around in this function to find the error.